projects/demo/src/modules/experimental/compass/compass.template.html

```diff
@@ -0,0 +1,77 @@
+<tui-doc-page
+    header="Compass"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <tui-doc-example
+            id="user-position"
+            description="It will be looks like user position on Google map"
+            heading="Compass mode"
+            [content]="example1"
+        >
+            <tui-compass-example-1></tui-compass-example-1>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-compass
+                [degrees]="degrees"
+                [style.color]="color"
+            ></tui-compass>
+        </tui-doc-demo>
+        <tui-doc-documentation>
+            <ng-template
+                documentationPropertyName="degrees"
+                documentationPropertyType="number"
+                [(documentationPropertyValue)]="degrees"
+            >
+                Defines a degree rotates
+            </ng-template>
+        </tui-doc-documentation>
+        <tui-doc-documentation heading="CSS customization">
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="style.color"
+                documentationPropertyType="string"
+                [documentationPropertyValues]="colorVariants"
+                [(documentationPropertyValue)]="color"
+            >
+                Custom color
+            </ng-template>
+        </tui-doc-documentation>
+    </ng-template>
+
+    <ng-template pageTab>
+        <ol class="b-demo-steps">
+            <li>
+                <p>
+                    Import
+                    <code>TuiCompassModule</code>
```

Comment:
nit: Let's not use the actual name of the module so it is easier to copy-paste the page for new components :) I use just `<p>Import module</p>`

Reply:
fixed